src/js/config.js

```diff
@@ -0,0 +1,10 @@
+// index.js
+function sayHello(name) {
+  if (name) {
+    console.log(`Hello, ${name}!`);
+  } else {
+    console.log('Hello, world!');
+  }
+}
+sayHello('Alice');
+sayHello();
```

Comment:
is this required?

Reply:
fixed https://github.com/Pranav-Patani/BookWise/pull/4/commits/29b8309f4516034c9aeae3e8251460c2c7a30231
It was a test code to check linting